api/types/server.go

```diff
@@ -652,25 +652,46 @@ func (s *ServerV2) MatchSearch(values []string) bool {
 	if s.GetKind() != KindNode {
 		return false
 	}
+Outer:
+	for _, searchV := range values {
+		for key, value := range s.Metadata.Labels {
+			if containsFold(key, searchV) || containsFold(value, searchV) {
```

Comment:
While we're doing perf improvements, could we replace `containsFold` with `github.com/charlievieth/strcase.Contains`?

Reply:
Good find E!